ui/src/app/utils/index.tsx

```diff
@@ -627,3 +630,7 @@ export function validateInputForMySQL(searchString: string): Array<string> {
   }
   return Array.from(inputErrors);
 }
+
+export function cdrVersion(workspace: Workspace, cdrs: CdrVersionListResponse): CdrVersion {
```

Comment:
can you add a verb to the function? I wasn't really sure what it did until I found the definition.